create-project-async/create_project_async.json

```diff
@@ -0,0 +1,38 @@
+{
+  "operationName": "LaunchTextProjectAsyncMutation",
```

Comment:
what should we do if we use json from row based / document based project example?

Reply:
don't need to do anything just replace the this content with script from pcw